client/me/account/main.jsx

```diff
@@ -243,6 +244,16 @@ const Account = localize(
 			this.setState( { [ name ]: value } );
 		},
 
+		handleSubmitButtonClick() {
+			const { unsavedSettings } = this.props.userSettings;
+			this.recordClickEvent( 'Save Account Settings Button' );
+			if ( has( unsavedSettings, colorSchemeKey ) ) {
+				this.props.recordTracksEvent( 'calypso_color_schemes_save', {
```

Comment:
The submit button could be active because a different setting was changed. Therefore we need to double check if the color scheme setting actually changed and only if that's the case track the new value as  `calypso_color_schemes_save`.

Reply:
Would it make sense to listen for [`change`](https://github.com/Automattic/wp-calypso/blob/9e758ceb22d98d7cab08e6be7714d337bc8a0971/client/lib/preferences/store.js#L78) on the flux store instead of hooking into a click event?  ([usage](https://github.com/Automattic/wp-calypso/blob/9e758ceb22d98d7cab08e6be7714d337bc8a0971/client/lib/preferences/README.md#usage))

We'd need to either provide the changed keys somehow or hack directly into the payload handler. What do you think?